datasets/peer_read/peer_read.py

```diff
@@ -0,0 +1,306 @@
+                            "creator": "",
+                        },
+                    )
+                    metadata["sections"] = [] if metadata["sections"] is None else metadata["sections"]
+                    metadata["sections"] = [
+                        {
+                            "heading": self._decode(section.get("heading", "")),
+                            "text": self._decode(section.get("text", "")),
+                        }
+                        for section in metadata["sections"]
+                    ]
+                    metadata["references"] = [] if metadata["references"] is None else metadata["references"]
+                    metadata["references"] = [
+                        {
+                            "title": reference.get("title", ""),
+                            "author": reference.get("author", []),
+                            "venue": reference.get("venue", ""),
+                            "citeRegEx": reference.get("citeRegEx", ""),
+                            "shortCiteRegEx": reference.get("shortCiteRegEx", ""),
+                            "year": reference.get("year", ""),
+                        }
+                        for reference in metadata["references"]
+                    ]
+                    metadata["referenceMentions"] = (
+                        [] if metadata["referenceMentions"] is None else metadata["referenceMentions"]
+                    )
+                    metadata["referenceMentions"] = [
+                        {
+                            "referenceID": self._decode(reference_mention.get("referenceID", "")),
+                            "context": self._decode(reference_mention.get("context", "")),
+                            "startOffset": self._decode(reference_mention.get("startOffset", "")),
+                            "endOffset": self._decode(reference_mention.get("endOffset", "")),
+                        }
+                        for reference_mention in metadata["referenceMentions"]
+                    ]
+
+                    yield id_, {
+                        "name": data.get("name", ""),
+                        "metadata": metadata,
+                    }
+                elif self.config.name == "reviews":
+                    yield id_, {
+                        "id": str(data.get("id", "")),
+                        "conference": str(data.get("conference", "")),
+                        "comments": str(data.get("comments", "")),
+                        "subjects": str(data.get("subjects", "")),
+                        "version": str(data.get("version", "")),
+                        "date_of_submission": str(data.get("date_of_submission", "")),
+                        "title": str(data.get("title", "")),
+                        "authors": data.get("authors", [])
+                        if isinstance(data.get("authors"), list)
+                        else [data.get("authors", "")],
```

Comment:
here you may allow a list with an empty string in it 
Is this expected ?

Reply:
I probably want to have just [] if there are no authors not [""] made the correction